system-addon/lib/TopSitesFeed.jsm

```diff
@@ -137,8 +136,12 @@ this.TopSitesFeed = class TopSitesFeed {
       data: this._getPinnedWithData()
     }));
   }
-  onAction(action) {
+  async onAction(action) {
     switch (action.type) {
+      case at.INIT:
```

Comment:
Also, if we block `NEW_TAB_LOAD`, the section would be collapsed until there were sites. (Although that early, other things aren't quite loaded yet either including strings !)

Check out this 10x slowed-down video of a debug build with this patch (and my load-correctly-on-startup patch):
https://ed.agadak.net/as/activity-stream.startup.10x.webm